src/PasteDialog.react.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable no-console */
```

Comment:
maybe remove this though

Reply:
It is merged @zsol . Could you please do the release ?

Thanks !